helm/keda/values.yaml

```diff
@@ -35,6 +35,24 @@ image:
 # Kubernetes cluster domain
 clusterDomain: cluster.local
 
+verticalPodAutoscaler:
+  keda:
+    enabled: false
```

Comment:
Would we ever have VPA disabled? Why not just render VPA unconditionally?

Reply:
Oh in our values that make sense yes

Reply:
I updated the values. Now vpa is enabled by default

Reply:
Did you update that in keda-upstream branches first then apply vendir?

Reply:
Oh no my bad. Let me fix it

Reply:
Done